src/cirrus/package.py

```diff
@@ -0,0 +1,388 @@
+  managed package, including build rules for the local venv
+
+"""
+import sys
+import os
+import contextlib
+import inspect
+import pystache
+import ConfigParser
+
+from argparse import ArgumentParser
+from cirrus.logger import get_logger
+from cirrus.utils import update_version
+import cirrus.templates
+
+from cirrus.git_tools import (
+    branch,
+    push,
+    get_tags,
+    tag_release,
+    commit_files_optional_push,
+    get_active_branch
+)
+
+
+DEFAULT_HISTORY_SENTINEL = "\nCIRRUS_HISTORY_SENTINEL\n"
+LOGGER = get_logger()
+
+
+def build_parser(argslist):
+    """
+    build CLI parser and process args
+    """
+    parser = ArgumentParser(
+        description=(
+            'git cirrus package command:'
+            ' initialises cirrus for an existing git repo'
+        )
+    )
+    parser.add_argument('command', nargs='?')
+
+    subparsers = parser.add_subparsers(dest='command')
+    init_command = subparsers.add_parser('init')
+    init_command.add_argument(
+        '--repo', '-r',
+        dest='repo',
+        default=os.getcwd()
+    )
+    init_command.add_argument(
+        '--source-dir', '-s',
+        help="source code directory within package, assumes top level dir if not set",
+        dest='source',
+        default=None
+    )
+    init_command.add_argument(
+        '--package', '-p',
+        help="name of package being bootstrapped",
+        dest='package',
+        required=True
+    )
+    init_command.add_argument(
+        '--version', '-v',
+        help="initial package version",
+        default='0.0.0',
+        dest='version'
+    )
+    init_command.add_argument(
+        '--organization', '-o',
+        dest='org',
+        default='ORGANIZATION HERE',
+    )
+    init_command.add_argument(
+        '--description', '-d',
+        dest='desc',
+        default='PACKAGE DESCRIPTION HERE'
+    )
+
+    init_command.add_argument(
+        '--templates',
+        help='template rules to include in MANIFEST',
+        nargs='+',
+        default=list()
+    )
+    init_command.add_argument(
+        '--version-file',
+        help='Version file, defaults to package __init__.py',
+        default=None
+    )
+    init_command.add_argument(
+        '--history-file',
+        help='changelog history filename',
+        default='HISTORY.md'
+    )
+    init_command.add_argument(
+        '--requirements',
+        help='requirements file for pip',
+        default='requirements.txt'
+    )
+    init_command.add_argument(
+        '--master-branch',
+        help='GitFlow master branch',
+        default='master',
+        dest='master'
+    )
+    init_command.add_argument(
+        '--develop-branch',
+        help='GitFlow develop branch',
+        default='develop',
+        dest='develop'
+    )
+    init_command.add_argument(
+        '--no-remote',
+        help='disable pushing changes to remote, commit locally only',
+        default=False,
+        action='store_true'
```

Comment:
The switch is false unless you specify the flag on the cli, so --no-remote flips that bool from False to True